CODEOWNERS

```diff
@@ -1,3 +1,3 @@
 * @mareklibra @jtomasek @rawagner
 
-/cypress @mareklibra @jtomasek @gamli75 @lalon4 @ukalifon @yobshans @cdvultur 
+/cypress @mareklibra @jtomasek @gamli75 @lalon4 @ukalifon @yobshans @cdvultur @nshidlin
```

Comment:
Can you please split this to separate PR?

Reply:
Yeah, slipped in from a test run. Removed